### DISCUSSION TITLE
What does clicking the clock icon do in the waybar?

### DISCUSSION THREAD

**Discussion by WGrav01** [prasanthrangan/hyprdots#1619] [Q&A]
When you click the clock icon in the waybar it becomes inverted, and when hovering over it it shows it as "activated" but there doesn't seem to be any documentation on it, so what is that functionality?
![240623_13h47m38s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/63617338/c5976e67-7fc0-42bf-a60f-c92aed68337a)

**Comment by mahaveergurjar** [comment]
idle inhibitor if you are active then the screen not off that is used for if you are watching movie or going that type of work so you don't want to  screen off the active this
